README.MD

```diff
@@ -1,7 +1,7 @@
 ﻿
 # AL Development for Dynamics 365 Business Central
  <!-- [![Released version](https://img.shields.io/badge/Release_version-0.12.15355-green.svg?style=flat-square)](https://mbs.microsoft.com/partnersource/global/deployment/downloads/product-releases/msdnav2018download) -->
- [![current preview version 0.15.18771](https://img.shields.io/badge/Current_Preview_Version-0.15.18771-orange.svg?style=flat-square)](https://github.com/Microsoft/AL/milestone/15) [![next preview update May 2018](https://img.shields.io/badge/Next_Preview_Update-May_2018-blue.svg?style=flat-square)](https://github.com/Microsoft/AL/milestone/16)
+ [![current preview version 0.16.19969](https://img.shields.io/badge/Current_Preview_Version-0.16.19969-orange.svg?style=flat-square)](https://github.com/Microsoft/AL/milestone/17?closed=1) [![next preview update August 2018](https://img.shields.io/badge/Next_Preview_Update-august_2018-blue.svg?style=flat-square)](https://github.com/Microsoft/AL/milestone/18)
```

Comment:
Milestone 17 is June update. Milestone 16 is May update ;)

Reply:
Hm, this confuses me a bit. I'm running the latest DevPreview (June 2018) which says it has version 0.16.19969 included. Is there a month offset between closing the milestone and releasing it into a DevPreview?

Reply:
That could very well be. But normally the milestone number with the version number always fits. Unfortunately I got a bit sloppy because I don't have access to the go ready program. But you can also check the titles of the milestones.

Reply:
Ok. All issues of milestone 17 = june 2018 say they are shipped. So, either the version numbers have a mismatch now, or the DevPreview of june has the AL version of may included. Let's wait if we get some official response.

Reply:
The June update is currently in the non-public Ready To Go preview.

The May update is currently in the public preview available on the Docker Hub

Reply:
There is no relation between vsix version 0.XXX and the milestone number.  The milestone number is just a sequential Id assigned by GitHub. June update is version 0.16.19969 and milestone link is milestone/17.